helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1191,6 +1200,11 @@ operator:
     collectorImage:
       repository: quay.io/signalfx/splunk-otel-collector
 
+# Should the CRDs be installed by a chart's crd/ directory or by using templates.
+operatorcrds:
+  # Should the CRDs be installed
+  install: true
```

Comment:
ah, maybe the default needs to be false unless operator is also enabled.

Reply:
Right. Please ensure that no crds are installed by default (auto-instrumentation disabled).